src/server/app/shared/plugins/auth.js

```diff
@@ -52,3 +58,20 @@ export async function generateToken(auth, username, password) {
   }
 };
 
+export async function validateToken(username, token) {
+  let user, secret, decodedToken, validation;
+  try {
+    user = await userStore.getUserByPhoneOrEmail(username);
```

Comment:
Do we expect to be passing in phone numbers here potentially?

Reply:
In the original design, I think it was set up so that users could use either as a privacy measure, but we can revisit that.